lib/audit/catalog_to_moab.rb

```diff
@@ -66,32 +66,33 @@ def check_catalog_version
     # TODO: anything special if preserved_copy.status is not OK_STATUS? - see #480
 
     unless online_moab_found?(druid, storage_dir)
+      update_status(PreservedCopy::ONLINE_MOAB_NOT_FOUND_STATUS)
       results.add_result(PreservedObjectHandlerResults::ONLINE_MOAB_DOES_NOT_EXIST)
```

Comment:
When I saw the response code and status together like this, I took a look at codes and statuses in general, and see that we have some codes and statuses that use `DOES_NOT_EXIST`, and some `NOT_FOUND`. My question is whether these are synonymous, and if so, do we want to decide on a unified convention? If so, I'd be for `NOT FOUND`.

Reply:
i don't have strong feelings about any of the above suggestions, would be fine with more consistency between names (and/or shorter names as per #372).

i agree with @tallenaz's point that using both "not found" and "does not exist" is a bit confusing, since we mean the same thing with both of those terms.  but i also agree with @ndushay's overarching goals (PreservedCopy model enum makes sense in its own context, devs of the future can easily see what's going on).

would also agree with @ndushay that this is outside the scope of this ticket, and that any follow on work along these lines could be done under #372 (or some other result class refactoring ticket).

Reply:
@tallenaz if you feel strongly enough about the naming mismatch, please make a ticket.  I'm okay with it dying off without further discussion, so it's up to you if you want to champion it.

Reply:
I agree the model object's status enum makes sense in its own context, and I'm happy for the devs of the future and their low cognitive overload, so for me this can fade away into the depths of github issue history.